internal/provisioner/cluster.go

```diff
@@ -234,6 +258,38 @@ func DeleteCluster(clusterID, s3StateStore string, logger log.FieldLogger) error
 	return nil
 }
 
+// waitForKubernetesReadyness will poll a given kubernetes cluster at a regular
```

Comment:
Should we push this into the kops interface?

Reply:
Yeah, we can do that for now. I had some thoughts as to other checks we may want to do that aren't kops-specific, but we can move this helper back out should we want to include some of that.